CONTRIBUTING.md

```diff
@@ -9,6 +9,9 @@
   * [sequelize model:create](https://www.npmjs.com/package/sequelize-cli)
   * [sequelize migrations:create](https://www.npmjs.com/package/sequelize-cli)
 
+* To create an api endpoint use jsonwebtoken to allow the user to verify routes.
+  * [jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken)
```

Comment:
Remove the explanatory text, and move this line into the README under the Back End section